tools/scripts/ansible/setup-playbook.yml

```diff
@@ -81,3 +133,19 @@
           - mosh
           - emacs-nox
           - byobu
+- hosts: letsencrypt
+  become: yes
+  vars_files:
+    - vars/main.yml
+    - local-vars/local.yml
+  tasks:
+    - name: Install certbot packages
+      apt:
+        pkg:
+          - python3-certbot-nginx
+    - name: setup certbot
+      command: sudo certbot --nginx  --agree-tos -m {{ certbot_admin_email }} -d {{ inventory_hostname }} --non-interactive --keep
```

Comment:
All this does is create the nginx config file.  Where do you generate the actual certificate, and how do you point the nginx config file to that certificate?

Reply:
For certbot config, cf. https://unicode-org.atlassian.net/browse/CLDR-13567

Reply:
The current production server uses /etc/nginx/sites-enabled/stproxy

I think certbot automatically generates the certificate if installed right.

Reply:
@sffc this does create the certificate and updates the config file, that's why the `python3-certbot-nginx` plugin was installed. https://cldr-smoke2.unicode.org was setup using only this playbook.